documentation/installation.md

```diff
@@ -10,24 +10,35 @@ system.
 
 Vagrant uses [VirtualBox](https://www.virtualbox.org/) to create isolated
 virtual machines with the developer environment set up. To provision and
-configure the developer environment as needed Vagrant uses
+configure the developer environment as needed, Vagrant uses
 [Ansible](https://www.ansible.com/).
 
-You will need to install all these tools before you can set up your development
+First, make sure that Intel virtualization is enabled in your BIOS / UEFI firmware.
+Then, you will need to install all these tools before you can set up your development
 environment:
 
 - [Vagrant Download](https://www.vagrantup.com/downloads.html)
 - [VirtualBox Download](https://www.virtualbox.org/wiki/Downloads)
 
-Once you have these tools installed, you will need to download the [Ansible
-playbooks](https://github.com/open-craft/ansible-playbooks) used to build the
-Vagrant instance.  If you haven't checked it out yet, you can clone it into the
-`deploy/` subdirectory using this command at the root of the repository:
+Install the plugin vagrant-vbguest:
+
+    vagrant plugin install vagrant-vbguest
+
+Once you have these tools installed, download the [Ocim repository]
+(https://github.com/open-craft/opencraft) if you have not already done it.
+
+    git clone https://github.com/open-craft/opencraft ocim
+    cd ocim
+
+Download also the [Ansible playbooks](https://github.com/open-craft/ansible-playbooks)
```

Comment:
```suggestion
Also download the [Ansible playbooks](https://github.com/open-craft/ansible-playbooks)
```

Reply:
is there any difference between "download also" and "also download" ?

Reply:
@alanoe, imho, while the latter is also grammatically correct, it sounds more colloquial and uncommon than the latter (my suggestion).

Reply:
Done